### ISSUE TITLE
Input - Move helper text out of aria live region

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1262]
CENG_3305

Docs: https://react-magma.cengage.com/version/4.3.0/api/input/

Success Criteria: https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Within the Error Message example, the "* required" text after the input is wrapped in an aria-live region using aria-live="polite". This is causing the text to be announced when the input is interacted with and when the form is submitted. 

This live region appears to be intended for the error message so the required text should be moved out of this.
